src/lib/Tab/Event/Subscriber/ConditionalTabSubscriber.php

```diff
@@ -51,7 +49,11 @@ public function onTabGroupInitialize(TabGroupEvent $tabGroupEvent)
         $tabGroup = $tabGroupEvent->getData();
         $tabGroupIdentifier = $tabGroupEvent->getData()->getIdentifier();
         $parameters = $tabGroupEvent->getParameters();
-        $tabs = $this->tabService->getTabGroup($tabGroupIdentifier)->getTabs();
+        try {
+            $tabs = $this->tabService->getTabGroup($tabGroupIdentifier)->getTabs();
+        } catch (\InvalidArgumentException $invalidArgumentException) {
```

Comment:
```suggestion
        } catch (\InvalidArgumentException $e) {
```

please also import `\InvalidArgumentException`.